3party/opening_hours/rules_evaluation.cpp

```diff
@@ -110,6 +112,9 @@ osmoh::RuleState ModifierToRuleState(osmoh::RuleSequence::Modifier const modifie
     case Modifier::Comment:
       return osmoh::RuleState::Unknown;
   }
+  std::cerr << "Unreachable\n";
+  std::abort();
+  return osmoh::RuleState::Unknown;
```

Comment:
This line should probably be removed. It has no effect (`std::abort` [has](https://en.cppreference.com/w/cpp/utility/program/abort) `[[noreturn]]`) and will likely trigger [-Wunreachable-code-return](https://clang.llvm.org/docs/DiagnosticsReference.html#wunreachable-code-return).